addons/sales_team/views/crm_team_member_views.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo><data>
+
+    <record id="crm_team_member_view_search" model="ir.ui.view">
+        <field name="name">crm.team.member.view.search</field>
+        <field name="model">crm.team.member</field>
+        <field name="arch" type="xml">
+            <search string="Sales Person">
+                <field name="name"/>
+                <field name="crm_team_id"/>
+                <separator/>
+                <filter name="archived" string="Archived" domain="[('active', '=', False)]"/>
+                <group expand="0" string="Group By">
+                    <filter string="Sales Team" name="groupby_crm_team_id" context="{'group_by': 'crm_team_id'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="crm_team_member_view_tree" model="ir.ui.view">
+        <field name="name">crm.team.member.view.tree</field>
+        <field name="model">crm.team.member</field>
+        <field name="arch" type="xml">
+            <tree string="Sales Men" sample="1">
+                <field name="crm_team_id"/>
+                <field name="user_id"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="crm_team_member_view_tree_from_team" model="ir.ui.view">
+        <field name="name">crm.team.member.view.tree.from.team</field>
+        <field name="model">crm.team.member</field>
+        <field name="inherit_id" ref="sales_team.crm_team_member_view_tree"/>
+        <field name="mode">primary</field>
+        <field name="priority">32</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='crm_team_id']" position="replace">
+            </xpath>
+        </field>
+    </record>
+
+    <record id="crm_team_member_view_kanban" model="ir.ui.view">
+        <field name="name">crm.team.member.view.kanban</field>
+        <field name="model">crm.team.member</field>
+        <field name="arch" type="xml">
+            <kanban quick_create="false"
+                create="true"
```

Comment:
Isn't it create True by default if you have necessary rights?

Reply:
Indeed